docs/ops/pooling/MaxPool_1.md

```diff
@@ -6,7 +6,79 @@
 
 **Short description**: [Reference](http://caffe.berkeleyvision.org/tutorial/layers/pooling.html)
```

Comment:
List just a reference is not good at all. Primarily, you need to describe the operation shortly like: `Performs max pooling on input`.